### DISCUSSION TITLE
GitHub Copilot: Solution 0 returned. finish reason: ["stop"]

### DISCUSSION THREAD

**Discussion by lotek4u** [community/community#45448] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I've been trying to get this to work since November without luck. This is for powershell. I can type out as many comments as I want, and it will make suggestions as I'm typing:
![image](https://user-images.githubusercontent.com/29106552/214893139-a943cda5-566e-492b-b3a0-abb88ec02e9e.png)

![image](https://user-images.githubusercontent.com/29106552/214892374-d767d184-c6f7-425e-97ea-3e84e57a8395.png)

But I never get back any code suggestions, only inline comment predictions (predictive text?)

**Comment by iamcjwilliams** [comment]
@lotekjunky Are you getting any options if you hover your mouse over your comments?

**Reply by lotek4u** [reply]
Sorry, I cancelled my account. It hasn't worked since November.

**Reply by TheHumanistX** [reply]
> @lotekjunky Are you getting any options if you hover your mouse over your comments?

I'm not. I am getting this same 'solution 0 returned. finish reason: [stop]'... and when I hover over the comment I get nothing. I just installed and started using.

**Reply by lotek4u** [reply]
@TheHumanistX I haven't used copilot in months because it doesn't work.
